### PR TITLE
feat(divider): prefix change

### PR DESCRIPTION
**Pull request by timrombergjakobsson**
**Describe pull-request**
Changes prefix for Divider.

**Solving issue**  
Fixes: [DTS-1635](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1635)

**How to test**  
1. Go to Storybook
2. Check in Components -> Divider
3. Check the components APIs and styles.

**Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

[DTS-1635]: https://tegel.atlassian.net/browse/DTS-1635?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: core/src/components/dropdown-v2/dropdown-filter.scss [hunk 1]
```diff
@@ -99,7 +99,7 @@
 
     tds-icon {
       margin-right: 16px;
-      z-index: calc(tds-z-index(dropdown) + 1);
+      z-index: tds-z-index(dropdown) + 1;
```
Comment (mJarsater) [inline core/src/components/dropdown-v2/dropdown-filter.scss:102]: Should this be part of this PR?

File: core/src/components/divider/divider.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Component, Prop, Host, h } from '@stencil/core';
 
 @Component({
-  tag: 'sdds-divider',
+  tag: 'tds-divider',
   styleUrl: 'divider-component.scss',
```
Comment (mJarsater) [inline core/src/components/divider/divider.tsx:5]: I think you can rename the scss file to divider.scss to conform with our code standards.

File: core/src/components/divider/divider-webcomponent.stories.tsx [hunk 3]
```diff
@@ -59,7 +59,7 @@ export default {
 const Template = ({ orientation, width, height }) =>
```
Comment (theJohnnyMe) [inline core/src/components/divider/divider-webcomponent.stories.tsx:59]: Maybe rename this file to just "divider.stories.tsx"

File: core/src/components/divider/divider-vars.scss [hunk 4]
```diff
@@ -1,14 +1,14 @@
 $border-types: top, right, bottom, left;
```
Comment (theJohnnyMe) [inline core/src/components/divider/divider-vars.scss:1]:
```suggestion

```
It seems it is not used, so maybe just delete it?

Comment (timrombergjakobsson) [reply core/src/components/divider/divider-vars.scss:1]: Sure, not really part of PR, but will fix!

**Review by mJarsater — APPROVED** [review]
Noice! 🎸

**Pull request merged**